models/webauthn_credential.go

```diff
@@ -0,0 +1,219 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/go-webauthn/webauthn/protocol"
+	"github.com/go-webauthn/webauthn/webauthn"
+)
+
+// WebAuthnCredential represents a single WebAuthn credential for a user
+type WebAuthnCredential struct {
+	ID              string                            `json:"id" gorm:"type:varchar(255)"`
+	Name            string                            `json:"name"`
+	CredentialID    []byte                            `json:"credential_id" gorm:"type:blob"`
+	PublicKey       []byte                            `json:"public_key" gorm:"type:blob"`
+	AttestationType string                            `json:"attestation_type"`
+	Transport       []protocol.AuthenticatorTransport `json:"transport" gorm:"-"`
+	TransportJSON   string                            `json:"-" gorm:"column:transport"`
+	Flags           struct {
```

Comment:
Can you explain what these flags are used for? Couldn't find them anywhere in the WebAuthn spec, but might just have overlooked them.